packages/legend-application-studio/src/stores/shared/PackageTreeUtils.ts

```diff
@@ -244,6 +249,17 @@ export const openNode = (
 ): PackageTreeNodeData | undefined => {
   let currentElement: PackageableElement | undefined = element;
   let openingNode: PackageTreeNodeData | undefined;
+  const rootId = getNullableFirstElement(treeData.rootIds);
+  let dependencyRootNode;
+  if (rootId !== undefined) {
+    dependencyRootNode = treeData.nodes.get(rootId);
+  }
+  if (
+    dependencyRootNode?.label ===
+    ExplorerTreeRootPackageLabel.PROJECT_DEPENDENCY
+  ) {
+    dependencyRootNode.isOpen = true;
+  }
```

Comment:
I think the bug arises from when it hits the dependencyNode in the while loop below and since the dependencyNode.package is null, it doesn't open that node. 
Can't we instead just open that node as well. Do a check after the while loop  to ensure that the currentElement Node is open even if the package is null. 

I don't like having dependency specific logic here.